capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './base_decorator'
+
+class CapitalizeDecorator < BaseDecorator
+  def correct_name
+    super.capitalize
+  end
```

Comment:
- [ ] Please fix this method as below to work the CapitalizeDecorator as expected.

```ruby
def correct_name
      @nameable.correct_name.capitalize
    end
```